content/en/blog/2025/expose-otel-collector-gateway-api/index.md

```diff
@@ -0,0 +1,445 @@
+---
+title: Securely Exposing Your  OTEL Collector in Kubernetes with Gateway API and mTLS
+date: 2025-04-6
+author: >
+  [Vipin Vijaykumar](https://github.com/vipinvkmenon)
+sig: OpenTelemetry JS
+cSpell:ignore: Danielson
+---
+
+# Securely Exposing Your  OTEL Collector in Kubernetes with Gateway API and mTLS
```

Comment:
You don't need to add your h1 title once again, this will be added based on the `title` above.